### ISSUE TITLE
Join TechEmpower Benchmarks!

### ISSUE THREAD

**Issue by AmirHmZz** [uNetworking/uWebSockets#1454]
It would be a great opportunity to see uWebaockets in [TechEmpower benchmark](https://github.com/TechEmpower/FrameworkBenchmarks)! IMHO it can leverages uWebaockets community and introduce it to more people.

**Comment by alexhultman** [comment]
TechEmpower benchmarks are not benchmarks for web servers. It's essentially a database benchmark, as all their tests involve database lookups.

**Comment by szmarczak** [comment]
Those *are* web server benchmarks as well, note the `plaintext` section.

**Comment by alexhultman** [comment]
Yep plaintext benchmarks are the only ones who qualify. But their execution is poor and tainted. So the overall conclusion is: we do our own benchmarks better, as we have done since 2016
